app/workers/sync_from_find.rb

```diff
@@ -3,7 +3,7 @@ class SyncFromFind
 
   def perform
     Rails.configuration.providers_to_sync[:codes].each do |code|
-      SyncProviderFromFind.call(provider_code: code)
+      SyncProviderFromFind.call(provider_code: code, sync_courses: true)
```

Comment:
This sets `sync_courses` according to the Rails provider config, which is great, but I can't tell when `SyncFromFind` is actually being called.

Reply:
I don't think it is being called at the moment. I had left it half thinking that we might want to manually sync the old way but that doesn't really make sense anymore. If there are no objections I think I should remove it completely?

Reply:
Another thing I've just realised is that `setup_local_dev_data` doesn't enable syncing on the test provider so I need to change that too.

Reply:
Yes please either use or remove.